README.md

```diff
@@ -40,7 +40,7 @@ You should use one of these three ways to import the AWS Mobile SDK but not mult
         $ sudo gem install cocoapods
         $ pod setup
 
-1. In your project directory (the directory where your `*.xcodeproj` file is), create a plain text file named `Podfile` (without any file extension) and add the lines below. Replace `YourTarget` with your actual target name.
+2. In your project directory (the directory where your `*.xcodeproj` file is), create a plain text file named `Podfile` (without any file extension) and add the lines below. Replace `YourTarget` with your actual target name.
```

Comment:
I pulled your changes and opened the README.md in the MacDown viewer. Still, the numbered lists aren't correct. It shows (1) instead of (2). Can you check this?

Reply:
Hey @kvasukib, I'm a bit confused. In MacDown viewer on my computer, things seem to be rendered as expected, with the numbering showing (2) in the correct place. I've attached an image here for reference. Is this not what you are seeing or is this numbering just not as expected?

<img width="1433" alt="screen shot 2017-12-26 at 3 38 24 pm" src="https://user-images.githubusercontent.com/8602661/34365949-0ed93f16-ea53-11e7-930f-a5d18ec78878.png">

Reply:
Can you check the numbering under "Getting Started with Swift" and "Getting Started with Objective-C" section? Attached the screenshot here.
![screenshot_readme](https://user-images.githubusercontent.com/15204269/34496851-0e50a55e-efb0-11e7-92d5-9a6a1e56abe0.png)

Reply:
Ah @kvasukib I see now, my apologies. I believe I've fixed this.

Reply:
Thanks, @jakerockland for fixing it. I have pulled the latest changes. Working on it.

Reply:
Sounds good! @kvasukib 👍

Reply:
We did a release. But we have a merge conflict. If you can fix it, I can merge your PR. Thanks!